Mathlib/Combinatorics/Additive/Energy.lean

```diff
@@ -45,6 +41,17 @@ def multiplicativeEnergy (s t : Finset α) : ℕ :=
 #align finset.multiplicative_energy Finset.multiplicativeEnergy
 #align finset.additive_energy Finset.additiveEnergy
 
+@[inherit_doc multiplicativeEnergy] notation3:max "Eₘ[" s ", " t "]" => multiplicativeEnergy s t
+@[inherit_doc additiveEnergy] notation3:max "E[" s ", " t "]" => additiveEnergy s t
+
+/-- The multiplicative energy of a finset `s` in a group is the number of quadruples
+`(a₁, a₂, b₁, b₂) ∈ s × s × s × s` such that `a₁ * b₁ = a₂ * b₂`. -/
+notation3:max "Eₘ[" s "]" => Finset.multiplicativeEnergy s s
+
+/-- The additive energy of two finsets `s` and `t` in a group is the number of quadruples
+`(a₁, a₂, b₁, b₂) ∈ s × s × s × s` such that `a₁ + b₁ = a₂ + b₂`. -/
+notation3:max "E[" s "]" => Finset.additiveEnergy s s
+
 @[to_additive additiveEnergy_mono]
 theorem multiplicativeEnergy_mono (hs : s₁ ⊆ s₂) (ht : t₁ ⊆ t₂) :
     multiplicativeEnergy s₁ t₁ ≤ multiplicativeEnergy s₂ t₂ :=
```

Comment:
Could you use the notations you have just introduced, even in preexisting lemmas? I promiss that I will review your PR anyway, so it's not a problem if it makes the file longer.